test/EmptyApp.FunctionalTests/FunctionalTest/TelemetryModuleWorkingEmptyAppTests.cs

```diff
@@ -7,19 +7,20 @@ public class TelemetryModuleWorkingEmptyAppTests : TelemetryTestsBase
     {
         private const string assemblyName = "EmptyApp.FunctionalTests";
 
-#if NET451
-
         [Fact]
         public void TestBasicDependencyPropertiesAfterRequestingBasicPage()
         {
+#if NET451
```

Comment:
I am not sure why, but visual studio had trouble recognizing these tests. It was not until I moved them in, that the test explorer identified these.